### PR TITLE
WDL size estimates

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: variant-caller/variant-caller-wdl/calculate_contamination.wdl [hunk 1]
```diff
@@ -17,11 +17,13 @@ workflow calulateDNAContamination {
...
-  Float reference_size = size(ref_fasta, "GB") + size(ref_fasta_index, "GB")
-  Float cram_size = size(input_cram_file, "GB") + size(input_crai_file, "GB")
+  Float? est_ref_size
+  Float reference_size = select_first([est_ref_size, 4.0])
+  Float? est_cram_size
+  Float cram_size = select_first([est_cram_size, 30.0])
 
   String? reference_genome_version
-  String reference_genome = select_first([reference_genome_version, 'hg38'])
+  String reference_genome = select_first([reference_genome_version, "hg38"])
```
Comment (ruchim) [inline variant-caller/variant-caller-wdl/calculate_contamination.wdl]: This was changed as the WDL syntax highlighter is more compatible with double-quoted strings

File: variant-caller/variant-caller-wdl-checker/topmed-variantcaller-checker.wdl [hunk 2]
```diff
@@ -10,7 +10,9 @@ task checkerTask {
   # Cromwell error from asking for 0 disk when the input is less than 1GB
   Int additional_disk = select_first([increase_disk_size, 20])
 
-  Float disk_size = size(inputTruthVCFFile, "GB") + size(inputTestVCFFile, "GB") + additional_disk
+  Float? est_input_vcf_size
+  Float input_vcf_size = select_first([est_input_vcf_size, 30.0])
```
Comment (ruchim) [inline variant-caller/variant-caller-wdl-checker/topmed-variantcaller-checker.wdl]: @wshands I wasn't sure what the max inputTestVCFFile could be, so just inflated it to be safe.

Comment (wshands) [reply variant-caller/variant-caller-wdl-checker/topmed-variantcaller-checker.wdl]: OK

**Review by wshands — CHANGES_REQUESTED** [review]
The changes look OK. Does it make sense to just comment out the disk size calculation so it is easy to restore it when FireCloud supports it for DOS ids? Also are we going to keep these changes in a feature branch? If not and you want to merge then please make the merge to develop instead of master.

**Comment by ruchim** [comment]
@wshands I think adding this to a feature branch is a great idea. It's probably the easiest way to  keep these changes isolated and easy to find for the Aug 1st demo. Is there a preferred name for this feature branch? In that case, I shouldn't need to comment out disk size calculations?

**Comment by wshands** [comment]
@ruchim I typically name feature branches 'feature/<some descriptive name>' for instance maybe it could be 'feature/pre-determine-disk-size' 
If you create a feature branch that ultimately will not be merged to develop or master then you can do whatever you want; so you could just remove the disk size calculations as you have done. 
Does that make sense?
(Then when exporting the WDL from Dockstore you just export the feature branch)

**Comment by wshands** [comment]
@ruchim Ruchi, I assume this pull request will be closed and the changes will remain in a feature branch?

**Pull request closed without merging**